lib/gitspindle/github.py

```diff
@@ -768,9 +765,10 @@ def issue(self, opts):
         for issue_no in opts['<issue>']:
             issue = repo.issue(issue_no)
             if issue:
+                pr = issue.pull_request()
                 print(wrap(issue.title.encode(sys.stdout.encoding, errors='backslashreplace').decode(sys.stdout.encoding), attr.bright, attr.underline))
                 print(issue.body.encode(sys.stdout.encoding, errors='backslashreplace').decode(sys.stdout.encoding))
-                print(issue.pull_request and issue.pull_request['html_url'] or issue.html_url)
+                print(pr and pr.html_url or issue.html_url)
```

Comment:
While we're at it, can we switch to the more modern `pr.html_url if pr else issue.html_url` syntax?

Reply:
done